scripts/docker-entrypoint.sh

```diff
@@ -126,14 +126,27 @@ else
    export SECRET_CASSANDRA_PASSWORD=${CASSANDRA_PASSWORD}
 fi
 
-/bin/dockerize -template ${CATALINA_HOME}/conf/Catalina/localhost/prweb.xml:${CATALINA_HOME}/conf/Catalina/localhost/prweb.xml
+/bin/dockerize -template ${CATALINA_HOME}/webapps/ROOT/index.html:${CATALINA_HOME}/webapps/ROOT/index.html
+
+appContextFileName=$(echo "${PEGA_APP_CONTEXT_PATH}"|sed 's/\//#/g')
+
+if [ ${PEGA_APP_CONTEXT_PATH} != "prweb" ]; then
+    # Move pega deployment out of webapps to avoid double deployment
+    cp -r ${PEGA_DEPLOYMENT_DIR}/* /opt/pega/prweb
```

Comment:
This seems like it would fail if the container was stopped and then restarted.

Is there a reason we can't just use a symlink so it appears to be in both places at once?

Reply:
When a container is restarted, wouldn't docker-entrypoint be run with a clean image (as if it were being started for the first time)?

Reply:
I thought about that, but I'm concerned that using a sym link might result in the app being deployed multiple times.

Reply:
I think you generally always get a clean filesystem in a Kubernetes context, but if you were to just stop and start the container via docker, then the filesystem will be preserved.

Reply:
Added logic to allow for container restarts